ui/pages/confirmations/components/simulation-details/useSimulationMetrics.ts

```diff
@@ -129,6 +140,8 @@ export function useSimulationMetrics({
     updateTransactionEventFragment,
     transactionId,
     JSON.stringify(params),
+    loading,
```

Comment:
Are these needed as dependencies since neither of them are referenced inside the `useEffect`?

They indirectly impact the `params` but that is already a dependency.

Reply:
Good catch.